compiler/tests/negative.ml

```diff
@@ -47,7 +47,7 @@ let config path =
         ("fail/typing/x86-64/non_inline_stack_arg.jazz", reset_warn_recoverable);
         ("fail/typing/x86-64/non_inline_stack_res.jazz", reset_warn_recoverable);
         ( "fail/typing/x86-64/write_constant_pointer_direct_array.jazz",
-          disable_warnings [ PedanticPretyping ] );
+          reset_warn_recoverable );
```

Comment:
Shouldn't we do the same for the next test?

Reply:
Of course no. These « pedandic » warnings should go away (from pretyping).